tox.ini

```diff
@@ -0,0 +1,49 @@
+[tox]
+envlist=py3
+skipsdist=True
+minversion=2.9.0
+skip_missing_interpreters=True
+
+[testenv]
+description=
+    run tests with pytest under {basepython} - the posargs specify the tests to run
+whitelist_externals=
+    mv
+    rm
+deps=
+    -r{toxinidir}/requirements.txt
+    asynctest
+    pytest-asyncio
+    pytest<4.0.0
+    pytest-cov>=2.5.1
+    pytest-html>=1.14.2
+    pytest-mock>=1.6.0
+    pytest-profiling>=1.2.6
+    pylint>=1.7.1
+    pylint-quotes>=0.1.5
+commands=
+    pytest -s \
+        --cov-report term \
+        --cov=aionetbox \
+        -p no:warnings \
+        {posargs}
+
+[testenv:lint]
+deps=
+    -r{toxinidir}/requirements.txt
+    pylint
+
+commands=
+    pylint --disable=C0111,C0301,R1720,W0107,C0103,R0913 aionetbox
+
```

Comment:
neither here nor there, but I've started using a different pattern for test / lint / formatting : https://github.com/vapor-ware/synse-server/blob/v3/staging/tox.ini

pylint was slow and some checks felt overly cumbersome, so I opted for tools that did simpler checking.

this tox config is still good and works, just making note of my latest rev on tox config, which I'm sure will eventually change again